test/gurka/test_gtfs.cc

```diff
@@ -0,0 +1,151 @@
+#include "gurka.h"
+#include "third_party/just_gtfs/include/just_gtfs/just_gtfs.h"
+
+#include "string.h"
+#include <gtest/gtest.h>
+
+using namespace gtfs;
+using namespace valhalla;
+
+#define tripOneID "bar"
+#define tripTwoID "barbar"
+#define stopOneID "foo"
+#define stopTwoID "foo2"
+#define shapeOneID "square"
+#define serviceOneID "bon"
+
+// test to write gtfs files
+TEST(GtfsExample, WriteGtfs) {
+
+  Feed feed;
+
+  // write agency.txt
+  struct Agency ttc {
+    .agency_id = "TTC", .agency_name = "Toronto Commission", .agency_url = "http://www.ttc.ca",
+    .agency_timezone = "America/Toronto"
+  };
+  feed.add_agency(ttc);
+  feed.write_agencies(("../build/test/data/gtfs_test/"));
```

Comment:
these are run from inside wherever your build folder is
```suggestion
  feed.write_agencies(("test/data/gtfs_test/"));
```